app/services/idv/proofing_components.rb

```diff
@@ -33,7 +33,11 @@ def document_type
     end
 
     def source_check
-      Idp::Constants::Vendors::AAMVA if idv_session.verify_info_step_complete?
+      if idv_session.resolution_successful
+        Idp::Constants::Vendors::AAMVA
+      elsif idv_session.resolution_successful == false
+        Idp::Constants::Vendors::AAMVA_UNSUPPORTED_JURISDICTION
+      end
```

Comment:
A/N: This seems less than ideal, but there doesn't seem to be any good method to pulling the supplied `vendor_name` out like I did in the `VerifyInfoConcern#add_proofing_components` method.

Reply:
I think there are 2 options here:

1. Add a value to the session to indicate whether we performed an AAMVA check and use that
2. Inspect the value of the issuing jurisdiction in `pii_from_user` or `pii_from_doc` and infer from that whether AAMVA was contacted

Option 1 may be slightly preferable since it stores the result of what the job actually does. The weakness is it involves adding another value to the session. As a result we'll need to break this into 2 deploys. One deploy to write the new value and a second to read and use it.